_includes/privacy-statement.md

```diff
@@ -1 +1,15 @@
-coming soon
\ No newline at end of file
+# Privacy Statement for the Metal³ Project
+
+As Metal3.io and most of the infrastructure of the Metal³ Project are currently hosted by Red Hat Inc., this site falls under the [Red Hat Privacy Policy](https://www.redhat.com/en/about/privacy-policy){:target="_blank"}.  All terms of that privacy policy apply to this site.  Should we change our hosting in the future, this Privacy Policy will be updated.
+
+## How to Contact Us
+
+If you have any questions about any of these practices or Metal³'s use of your personal information, please feel free to [contact us](mailto:privacy@metal3.io) or [file an Issue](https://github.com/metal3-io/metal3-io.github.io/issues){:target="_blank"} in our Github repo.
+
+Metal³ will work with you to resolve any concerns you may have about this Statement.
+
+## Changes to this Privacy Statement
+
+Metal³ reserves the right to change this policy from time to time. If we do make changes, the revised Privacy Statement will be posted on this site. A notice will be posted on our blog and/or mailing lists whenever this privacy statement is changed in a material way.
+
+This Privacy Statement was last amended on September 6, 2018.
```

Comment:
It should be updated to 29th August as I did for /privacy to match when we sent the email to mailing list